schema/deploy/shipping/views.sql

```diff
@@ -2131,4 +2147,170 @@ grant select
+create or replace view shipping.scan_demographics_v1 as
+
+    select
+        encountered_week,
+        sex,
+        age_range_fine,
+        age_range_fine_lower,
+        age_range_fine_upper,
+        race,
+        hispanic_or_latino,
+        income,
+        puma
+    from shipping.scan_encounters_v1
+;
+
+comment on view shipping.scan_demographics_v1 is
+  'A view of basic demographic data from the SCAN project for Power BI dashboards.';
+
+revoke all
+    on shipping.scan_demographics_v1
+  from "power-bi-exporter";
+
+grant select
+    on shipping.scan_demographics_v1
+    to "power-bi-exporter";
+
+
+create or replace view shipping.scan_hcov19_positives_v1 as
+
+    with hcov19_presence_absence as (
+        -- Collapse potentially multiple hCoV-19 results
+        select distinct on (sample_id)
+            sample.identifier as sample,
+            presence_absence_id,
+            pa.present as hcov19_present,
+            pa.created::date as hcov19_result_release_date
+        from
+            warehouse.presence_absence as pa
+            join warehouse.target using (target_id)
+            join warehouse.organism using (organism_id)
+            join warehouse.sample using (sample_id)
+        where
+            organism.lineage <@ 'Human_coronavirus.2019'
+            and pa.details @> '{"assay_type": "Clia"}'
+            and not control
+            -- We shouldn't be receiving these results from Samplify, but they
+            -- sometimes sneak in. Be sure to block them from this view so as
+            -- to not return inaccurate results to participants.
+            and target.identifier not in ('COVID-19_Orf1b', 'COVID-19-S_gene')
+        /*
+          Keep only the most recent push. According to Lea, samples are only
+          retested if there is a failed result. A positive, negative, or
+          indeterminate result would not be retested.
+
+          https://seattle-flu-study.slack.com/archives/CV1E2BC8N/p1584570226450500?thread_ts=1584569401.449800&cid=CV1E2BC8N
+        */
+        order by
+            sample_id, presence_absence_id desc
+    ),
+
+    scan_hcov19_results as (
+      select
+          case
+              when hcov19_present is true then 'positive'
+              when hcov19_present is false then 'negative'
+              when hcov19_present is null then 'inconclusive'
+          end as hcov19_result,
+          hcov19_result_release_date,
+          case
+              when puma in ('5310901', '5310902') then 'yakima'
+              when puma in ('5311701', '5311702', '5311703',
+                            '5311704', '5311705', '5311706') then 'snohomish'
+              when puma in ('5311601', '5311602', '5311603', '5311604',
+                            '5311605', '5311606', '5311607', '5311608',
+                            '5311609', '5311610', '5311611', '5311612',
+                            '5311613', '5311614', '5311615', '5311616') then 'king'
+          end as county
```

Comment:
You could add an explicit "else null" to show that null is possible. You'll still get nulls without the line, of course, but adding it could make the SQL easier to grock.